src/encoded/homepage.py

```diff
@@ -0,0 +1,166 @@
+from pyramid.view import view_config
+from snovault.util import debug_log
+from dcicutils.misc_utils import ignored
+from snovault.search.search import (
+    search
+)
+from snovault.search.search_utils import make_search_subreq
+from urllib.parse import urlencode
+import concurrent.futures
+
+
+import structlog
+
+
+log = structlog.getLogger(__name__)
+
+
+def includeme(config):
+    config.add_route('home', '/home')
+    config.scan(__name__)
+
+
+def make_concurrent_search_requests(search_helpers):
+    """ Execute multiple search functions concurrently using a thread pool (since this is I/O bound). """
+    results = [-1] * len(search_helpers)  # watch out for -1 counts as indicative of an error
```

Comment:
Are these -1 values being interpreted anywhere as indicative of an error currently?